Gemfile.lock

```diff
@@ -448,4 +506,4 @@ DEPENDENCIES
   wirble
 
 BUNDLED WITH
-   1.13.6
+   1.15.1
```

Comment:
I'm not sure about all the changes on the `Gemfile.lock`. Are you sure you only run `bundle install grape`, and not `bundle update` ? I'd like to keep the changes on versions as minimal as possible, since we will have to maintain them 😉